### DISCUSSION TITLE
How can the community help with PRs in triage?

### DISCUSSION THREAD

**Discussion by thixotropist** [NationalSecurityAgency/ghidra#6020] [General]
There are about 69 PR's with the Triage status, out of about 180 open PRs.  Can the Ghidra development team suggest any ways for the community to help advance or resolve this status?  Your `Status: Triage` text says that 'information is being gathered'.  If the needed information is public, we can likely help.

If triage is blocked waiting for proofs that *internal* projects do not fail if the PR is accepted, that's a different matter.

**Comment by ryanmkurtz** [comment]
For PR's, Triage means that the ticket has been assigned to an internal team member to review.  Part of this review is determining the priority of actually doing the review against all of their other tickets, which might be a substantial amount for some teammates.  Often times properly reviewing a PR can even be more time consuming than implementing it internally, so that has to be factored in too. 

IMO, the best way to move a PR along is for someone to "poke" the ticket, saying how useful it would be to include.  Of course, this may ultimately lead to it being rejected faster for one reason or another too.

**Comment by thixotropist** [comment]
Thanks for the comments.  Many of the open Triage PRs involve processor-specific changes, which your internal team members may find more difficult or time consuming to review.  You might consider either a slightly more open review process or maybe some long-lived processor-specific forks for the public community to work up PRs that are easier to review.

**Reply by ryanmkurtz** [reply]
Thank you, i'll pass these suggestions on to the team.

**Reply by thixotropist** [reply]
Sounds good.  As an example, the RISCV instruction set extensions PR #5778 probably won't be too broadly useful until mid 2024 - but at that point, it will be too big to easily review without a full RISCV toolchain environment.  I expect that it will be more useful than the existing RISCV language definitions for those newer chips.

The longest lead-time aspect of the review might be polling your users - both internal and external - whether they have a problem deprecating RISCV support for unratified and abandoned RISCV processor extensions in Ghidra 12.
